test/e2e/fixtures/releases/test/fail-no-rollback.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: podinfo-helm-repository
+  namespace: demo
+spec:
+  releaseName: podinfo-helm-repository
+  timeout: 15
+  test:
+    enable: true
+    timeout: 30
+  chart:
+    repository: https://stefanprodan.github.io/podinfo
+    name: podinfo
+    version: 3.2.0
+  values:
+    replicaCount: 2
+    faults:
+      testFail: true
```

Comment:
@stefanprodan this depends on https://github.com/stefanprodan/podinfo/pull/60

Reply:
You can use podinfo chart v3.2.4. Thanks for adding these faults to the chart 👍

Reply:
Thanks for the quick release!

Reply:
There was a bug in the tests yaml, I've fixed in `3.3.1`.